scripts/_discovery.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
+import argparse
+import argcomplete
+from tuda_workspace_scripts.print import *
+from tuda_workspace_scripts.robots import *
+from tuda_workspace_scripts.discovery import *
+from tuda_workspace_scripts.print import print_warn
+
+class DiscoveryServerChoicesCompleter:
+    def __call__(self, prefix, parsed_args, **kwargs):
+        complete_args = [var for var in load_robots().keys()]
+        # allowing a local server with ID 0 and default port
+        complete_args.append("local_server")
```

Comment:
Why is this called local_server instead of localhost?

Reply:
The idea was to indicate that a local server still needs to be started. It can be changed to localhost.